internal/pkg/encoding/json/schema/schema.go

```diff
@@ -28,7 +29,10 @@ func ValidateSchemas(objects model.ObjectStates) error {
 			return err
 		}
 
-		if err := ValidateConfig(component, config.Local); err != nil {
+		var schemaErr *SchemaError
+		if err := ValidateConfig(component, config.Local); errors.As(err, &schemaErr) {
+			logger.Warn(errors.PrefixErrorf(schemaErr.Unwrap(), `config JSON schema of the component "%s" is invalid, please contact support`, component.ID))
```

Comment:
A z nevalidnej schemy som spravil warning, sem tam sa to stane, uzivatel to nevie sam fixnut a zbyzocne ho blokovat pri praci s CLI.